guides/04_integrating-other-frameworks/Gradio-and_Comet.md

```diff
@@ -0,0 +1,273 @@
+# Using Gradio and Comet
```

Comment:
nit: Call this file `Gradio-and-Comet.md` (no underscore) otherwise the underscore will show up as the guide name

![image](https://user-images.githubusercontent.com/41651716/215872644-86f89e66-1d6e-4c15-9c9c-2e4ae99caec1.png)